cms/static/js/views/course_video_settings.js

```diff
@@ -503,6 +670,66 @@ function($, Backbone, _, gettext, moment, HtmlUtils, StringUtils, TranscriptSett
             }
         },
 
+        changeOrganizationCredentials: function(event) {
+            var self = this;
+            event.preventDefault();
+
+            ViewUtils.confirmThenRunOperation(
```

Comment:
@srpearce Can you look at these strings. These are shown when user clicks on "Change" button